_services_projects/optimistic-futures.md

```diff
@@ -6,7 +6,7 @@ permalink: /projects/optimistic-futures/
 redirect_from: /projects/optimistic-futures/
 excerpt: Speculating how government, public institutions and society could be resilient and prosperous in the future.
 image: /assets/img/projects/optimistic-futures.png
-image_accessibility: 
+image_accessibility: Image of Lego figures.
```

Comment:
Can you add what the Lego figurines are doing? So the blind user gets all the same info a sighted user does.